### DISCUSSION TITLE
Development tools

### DISCUSSION THREAD

**Discussion by wschell** [jamulussoftware/jamulus#1669] [Technical Support & Help] [answered]
What tools do I need to contribute to bugfinding/-fixing, translation (german), maybe new features. Coming from a windows background I'm used to eclipse (Java, Cobol), visual studio (C#, Cpp), and Delphi environment. But now I want to start developing with linux (unbuntu 20.4.2). What integrated development tool would you recommend, including compiling, building, source-code-debugging? 
I'm used to linux shell and vi, but have only a rough overview of git. 
Thank you for any hints.
Walter

**Comment by jujudusud** [comment, marked as answer]
For app translation of Jamulus, you have to use Qt linguist.

**Comment by dcorson-ticino-com** [comment]
I simply use Qt Creator under Win10.

**Reply by wschell** [reply]
Can I use the most recent version (6.1.0) or should I use 5.15.2?
EDIT: Oh I see now - it is the qt version , not the creator version. Nevertheless - what qt-version would be required?

**Reply by dcorson-ticino-com** [reply]
I am using Qt Creator 4.14.0 which has Qt 5.15.2
I don't know if that is recommended, but it works.

**Reply by wschell** [reply]
Thank you - than I will start with this configuration.
